sleap/gui/app.py

```diff
@@ -1659,14 +1724,14 @@ def _show_learning_dialog(self, mode: str):
 
         Returns:
             None.
+
         """
         from sleap.gui.learning.dialog import LearningDialog
 
         if "inference" in self.overlays:
             QMessageBox(
-                text="In order to use this function you must first quit and "
-                "re-open SLEAP to release resources used by visualizing "
-                "model outputs."
+                text="In order to use this function you must uncheck "
+                "'Visualize Model Outputs' to release resources used."
```

Comment:
Flagging this @talmo to double check that this is true. I just `self.overlays.pop("inference", None)` to free resources. I've tested this (not extensively) and haven't run into any problems.

Reply:
Does GPU memory get released? This might be from loading the model.